libs/header/test/testing.go

```diff
@@ -41,7 +41,7 @@ func (d *DummyHeader) IsZero() bool {
 }
 
 func (d *DummyHeader) ChainID() string {
-	return d.Raw.ChainID
+	return "private"
```

Comment:
p2p.Private

Reply:
this is lib. It should not know anything about p2p package I think.

Reply:
oh fair point